var/spack/repos/builtin/packages/valgrind/package.py

```diff
@@ -22,8 +22,8 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
-
 from spack import *
+import sys
 
 
 class Valgrind(Package):
```

Comment:
Since you're making such large changes, do you think its worth making this an `AutoToolsPackage` now?